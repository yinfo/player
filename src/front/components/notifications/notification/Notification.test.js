// @flow

import React from 'react';
import { shallow } from 'enzyme';
import Notification from './Notification';

describe('Notification component', () => {
  const props = {
    type: 'info',
  };

  it('renders as expected', () => {
    const component = shallow(
      <Notification {...props}>
        <p>a notification</p>
      </Notification>,
    );
    expect(component).toMatchSnapshot();
  });
});
