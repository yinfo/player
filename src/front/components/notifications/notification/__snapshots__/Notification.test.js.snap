// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification component renders as expected 1`] = `ShallowWrapper {}`;
